pkgs/top-level/linux-kernels.nix

```diff
@@ -193,7 +188,7 @@ in {
        else testing;
 
     linux_testing_bcachefs = callPackage ../os-specific/linux/kernel/linux-testing-bcachefs.nix rec {
-      kernel = linux_5_19;
+      kernel = linux_6_0;
```

Comment:
not sure if this is the right move. Seems like most a lot of broken builds:
```
https://github.com/NixOS/nixpkgs/pull/198992

54 packages marked as broken and skipped:
linuxKernel.packages.linux_testing_bcachefs.akvcam linuxKernel.packages.linux_testing_bcachefs.ax99100 linuxKernel.packages.linux_testing_bcachefs.can-isotp linuxKernel.packages.linux_testing_bcachefs.ena linuxKernel.packages.linux_testing_bcachefs.intel-speed-select linuxKernel.packages.linux_testing_bcachefs.ixgbevf linuxKernel.packages.linux_testing_bcachefs.kvdo linuxKernel.packages.linux_testing_bcachefs.mwprocapture linuxKernel.packages.linux_testing_bcachefs.mxu11x0 linuxKernel.packages.linux_testing_bcachefs.ndiswrapper linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy340 linuxKernel.packages.linux_testing_bcachefs.nvidia_x11_legacy390 linuxKernel.packages.linux_testing_bcachefs.nvidiabl linuxKernel.packages.linux_testing_bcachefs.openafs linuxKernel.packages.linux_testing_bcachefs.openafs_1_8 linuxKernel.packages.linux_testing_bcachefs.phc-intel linuxKernel.packages.linux_testing_bcachefs.r8125 linuxKernel.packages.linux_testing_bcachefs.r8168 linuxKernel.packages.linux_testing_bcachefs.rtl8188eus-aircrack linuxKernel.packages.linux_testing_bcachefs.rtl8192eu linuxKernel.packages.linux_testing_bcachefs.rtl8723bs linuxKernel.packages.linux_testing_bcachefs.rtl88xxau-aircrack linuxKernel.packages.linux_testing_bcachefs.sch_cake linuxKernel.packages.linux_testing_bcachefs.tbs linuxKernel.packages.linux_testing_bcachefs.vmware linuxKernel.packages.linux_testing_bcachefs.zfs linuxKernel.packages.linux_testing_bcachefs.zfsStable linuxPackages_testing_bcachefs.akvcam linuxPackages_testing_bcachefs.ax99100 linuxPackages_testing_bcachefs.can-isotp linuxPackages_testing_bcachefs.ena linuxPackages_testing_bcachefs.intel-speed-select linuxPackages_testing_bcachefs.ixgbevf linuxPackages_testing_bcachefs.kvdo linuxPackages_testing_bcachefs.mwprocapture linuxPackages_testing_bcachefs.mxu11x0 linuxPackages_testing_bcachefs.ndiswrapper linuxPackages_testing_bcachefs.nvidia_x11_legacy340 linuxPackages_testing_bcachefs.nvidia_x11_legacy390 linuxPackages_testing_bcachefs.nvidiabl linuxPackages_testing_bcachefs.openafs linuxPackages_testing_bcachefs.openafs_1_8 linuxPackages_testing_bcachefs.phc-intel linuxPackages_testing_bcachefs.r8125 linuxPackages_testing_bcachefs.r8168 linuxPackages_testing_bcachefs.rtl8188eus-aircrack linuxPackages_testing_bcachefs.rtl8192eu linuxPackages_testing_bcachefs.rtl8723bs linuxPackages_testing_bcachefs.rtl88xxau-aircrack linuxPackages_testing_bcachefs.sch_cake linuxPackages_testing_bcachefs.tbs linuxPackages_testing_bcachefs.vmware linuxPackages_testing_bcachefs.zfs linuxPackages_testing_bcachefs.zfsStable

73 packages failed to build:
linux_testing_bcachefs linuxPackages_testing_bcachefs.acpi_call linuxPackages_testing_bcachefs.apfs linuxPackages_testing_bcachefs.asus-ec-sensors linuxPackages_testing_bcachefs.asus-wmi-sensors linuxPackages_testing_bcachefs.batman_adv linuxPackages_testing_bcachefs.bbswitch linuxPackages_testing_bcachefs.broadcom_sta linuxPackages_testing_bcachefs.chipsec linuxPackages_testing_bcachefs.cryptodev linuxPackages_testing_bcachefs.ddcci-driver linuxPackages_testing_bcachefs.dddvb linuxPackages_testing_bcachefs.digimend linuxPackages_testing_bcachefs.dpdk linuxPackages_testing_bcachefs.dpdk-kmods linuxPackages_testing_bcachefs.evdi linuxPackages_testing_bcachefs.facetimehd linuxPackages_testing_bcachefs.fwts-efi-runtime linuxPackages_testing_bcachefs.gcadapter-oc-kmod linuxPackages_testing_bcachefs.hid-ite8291r3 linuxPackages_testing_bcachefs.hid-nintendo linuxPackages_testing_bcachefs.isgx linuxPackages_testing_bcachefs.it87 linuxPackages_testing_bcachefs.ithc linuxPackages_testing_bcachefs.jool linuxPackages_testing_bcachefs.kvmfr linuxPackages_testing_bcachefs.liquidtux linuxPackages_testing_bcachefs.lttng-modules linuxPackages_testing_bcachefs.mba6x_bl linuxPackages_testing_bcachefs.mbp2018-bridge-drv linuxPackages_testing_bcachefs.netatop linuxPackages_testing_bcachefs.new-lg4ff linuxPackages_testing_bcachefs.nvidia_x11 linuxPackages_testing_bcachefs.nvidia_x11_beta_open linuxPackages_testing_bcachefs.nvidia_x11_legacy470 linuxPackages_testing_bcachefs.nvidia_x11_production linuxPackages_testing_bcachefs.nvidia_x11_production_open linuxPackages_testing_bcachefs.nvidia_x11_vulkan_beta linuxPackages_testing_bcachefs.nvidia_x11_vulkan_beta_open linuxPackages_testing_bcachefs.openrazer linuxPackages_testing_bcachefs.prl-tools linuxPackages_testing_bcachefs.qc71_laptop linuxPackages_testing_bcachefs.rr-zen_workaround linuxPackages_testing_bcachefs.rtl8189es linuxPackages_testing_bcachefs.rtl8189fs linuxPackages_testing_bcachefs.rtl8812au linuxPackages_testing_bcachefs.rtl8814au linuxPackages_testing_bcachefs.rtl8821au linuxPackages_testing_bcachefs.rtl8821ce linuxPackages_testing_bcachefs.rtl8821cu linuxPackages_testing_bcachefs.rtl88x2bu linuxPackages_testing_bcachefs.rtw88 linuxPackages_testing_bcachefs.sysdig linuxPackages_testing_bcachefs.system76 linuxPackages_testing_bcachefs.system76-acpi linuxPackages_testing_bcachefs.system76-io linuxPackages_testing_bcachefs.systemtap linuxPackages_testing_bcachefs.tp_smapi linuxPackages_testing_bcachefs.tuxedo-keyboard linuxPackages_testing_bcachefs.usbip linuxPackages_testing_bcachefs.v4l2loopback linuxPackages_testing_bcachefs.v86d linuxPackages_testing_bcachefs.veikk-linux-driver linuxPackages_testing_bcachefs.vendor-reset linuxPackages_testing_bcachefs.vhba linuxPackages_testing_bcachefs.virtio_vmmci linuxPackages_testing_bcachefs.virtualbox linuxPackages_testing_bcachefs.virtualboxGuestAdditions linuxPackages_testing_bcachefs.vmm_clock linuxPackages_testing_bcachefs.xone linuxPackages_testing_bcachefs.xpadneo linuxPackages_testing_bcachefs.zenpower linuxPackages_testing_bcachefs.zfsUnstable

7 packages built:
linuxPackages_testing_bcachefs.cpupower linuxPackages_testing_bcachefs.hyperv-daemons linuxPackages_testing_bcachefs.perf linuxPackages_testing_bcachefs.ply linuxPackages_testing_bcachefs.tmon linuxPackages_testing_bcachefs.turbostat linuxPackages_testing_bcachefs.x86_energy_perf_policy
```

Reply:
cc. @davidak @Madouura

Reply:
Fixed by updating the patches